resources/metadata/detect.yaml

```diff
@@ -289,6 +289,16 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: customEnvironmentVariables
+        description:
+          "A list of environment variables which can be set to prepare the environment to run a BlackDuck scan. Specifically used to set the BlackDuck environment variables which control the scanning script"
+        type: "[]string"
+        default:
+          - "DETECT_LATEST_RELEASE_VERSION=6.8.0"
```

Comment:
Is this something that should go into the internal defaults?

Reply:
Yes, The value is only default internally for now. The parameter however is can be used by everyone to set custom environment variables.

Reply:
So we should introduce only the parameter here without a default and add the default internally.